pattern-distributed/confs/sp-worker/conf/deployment.yaml

```diff
@@ -346,21 +378,180 @@ wso2.datasources:
   - name: WSO2_PERMISSIONS_DB
     description: The datasource used for permission feature
     jndiConfig:
-        name: jdbc/PERMISSION_DB
-        useJndiReference: true
+      name: jdbc/PERMISSION_DB
+      useJndiReference: true
     definition:
         type: RDBMS
         configuration:
           jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/${sys:wso2.runtime}/database/PERMISSION_DB;IFEXISTS=TRUE;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000;MVCC=TRUE'
           username: wso2carbon
           password: wso2carbon
           driverClassName: org.h2.Driver
-          maxPoolSize: 50
+          maxPoolSize: 10
           idleTimeout: 60000
           connectionTestQuery: SELECT 1
           validationTimeout: 30000
           isAutoCommit: false
 
+  - name: HTTP_ANALYTICS_DB
```

Comment:
We can remove these datasources since they are not used for distributed deployment.